modules/tables/table_NUM_by_JayPrakash.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
No need for the Doctype declaration. We can't have two, your goal is to make a module that would be imported into any project. Thanks for understanding. Pls check module->buttons to see some button moduels that have been accepted before now.